frontend/src/components/atoms/TransactionEntry.tsx

```diff
@@ -154,5 +156,6 @@ export default styled(TransactionEntry)`
   &:hover {
     padding-left: 0.3em;
     border-left: 2px solid black;
+    background-color: #ebeff2;
```

Comment:
Should this perhaps be in the `theme` object if we're planning to use this as a highlight colour in the future? If not, just leave it as it is :)

Reply:
I will add it to the theme